test/fixtures/controller.bar/borderColor/indexable.js

```diff
@@ -0,0 +1,54 @@
+module.exports = {
```

Comment:
This looks a lot like `backgroundColor/indexable.js`. Is there a possibility of sharing code?

Reply:
It's not possible because it's not a real `module.exports` and I think it would require more work to be able to import a file from these fixture `.js` files.